tsconfig.json

```diff
@@ -1,5 +1,7 @@
 {
   "compilerOptions": {
-    "experimentalDecorators": true
+    "target": "ES5",
```

Comment:
This combination of es5 plus experimental decorators seem to do the trick locally.

A bit of a shame, I'd rather not have these at all.

Reply:
@WillForrestMindBridge see [my pr](https://github.com/GameBridgeAI/ts_serialize/pull/153/files#diff-a5de3e5871ffcc383a2294845bd3df25d3eeff6c29ad46e3a396577c413bf357) I don't think `target: ES5` is needed, we needed to tell the deno engine in vscode where the tsconfig file is located.